src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -485,10 +485,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             if from_flax:
                 ignore_patterns = ["*.bin", "*.safetensors"]
                 allow_patterns += [
-                    SCHEDULER_CONFIG_NAME,
-                    CONFIG_NAME,
                     FLAX_WEIGHTS_NAME,
-                    cls.config_name,
                 ]
```

Comment:
redundant allow patterns already added to array here: https://github.com/huggingface/diffusers/blob/cc3f485376ddf3bd76d877426d27cdeb583ec069/src/diffusers/pipelines/pipeline_utils.py#L474-L480